docs/user/reference/general/register-custom-pings.md

```diff
@@ -72,4 +77,8 @@ pings = load_pings(resource_filename(__name__, "pings.yaml"))
 
 </div>
 
+<div data-lang="Rust" class="tab"></div>
+<div data-lang="Javascript" class="tab"></div>
+<div data-lang="Firefox Desktop" class="tab"></div>
```

Comment:
Can we file bugs for the missing docs? The only ones that _actually_ don't have this API is Javascript and Firefox, right?

Reply:
I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1714942